src/openrct2/world/Sprite.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*****************************************************************************
+/*****************************************************************************
```

Comment:
BOM

Reply:
It appears to be on the left side, this reverts it apparently.

Reply:
Looks like you're right, it was introduced in efa1db826e2789e3d842995d5ced1ffb5e0a5a9c.